pages/common/finger.md

```diff
@@ -0,0 +1,23 @@
+# finger
+
+> User information lookup program.
+
+- Display information about currently logged in users:
+
+`finger`
+
+- Display information about a specific user:
+
+`finger {{username}}`
+
+- Display the user's login name, real name, terminal name and write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number:
+
+`finger -s`
+
+- Produce multi-line format displaying same information as `-s` as well as user's home directory, home phone number, login shell, mail status as well as contents of files ".plan", ".project", ".pgpkey", and ".foward" in their home directory:
```

Comment:
Produce _a_ multi-line _output_ format

Reply:
differs from the man page description, but I agree that it's clearer this way.

Reply:
Yes, the whole point is to differ from the man page. :smile: